### ISSUE TITLE
Spending this balance will need significant bitcoin network fees

### ISSUE THREAD

**Issue by aradell** [bitpay/copay#6595]
Hello,i have a problem with copay and I can not withdraw.Settings:super economy.
I tried all the withdraw variants and 
say:error insufficient funds.

I have this message in balance: spending this balance will need significant bitcoin network fees
How can I resolve this?
Thx

**Comment by JDonadio** [comment]
See https://github.com/bitpay/copay/issues/6562#issuecomment-322762885
https://github.com/bitpay/copay/wiki/COPAY---FAQ#copay-requires-high-fees-for-low-amounts-transactions

**Comment by Rob6179** [comment]
I have this same issue. I have .002BTC in my wallet as a result of two .001BTC transfers. The value of this is 8.15USD. On super economy it tells me that it will need significant network fees. 

My understanding was that the fees should be maybe a dollar at the most depending on data transaction size . Are the network fees really as much as 8 USD for a single transaction? 

Is there any effective way I can get my funds out if I don't have any other planned deposits?

Thanks

**Comment by MelisaSingh** [comment]
I know nothing about code. 

I have read and re-read through this thread trying to wrap my head around this issue. 

I have 7,065 bits (currently 30.79 USD) in a Copay wallet. The only way I have ever gotten bitcoin into this wallet was from withdrawals from Coinbase in amounts over $15 USD each time. When I go to transfer it says "Not enough funds for fee".  Surely $30 is more than enough to send, say $20 plus fees. 

I saw that JDonadio referenced this thread: https://github.com/bitpay/copay/wiki/COPAY---FAQ#copay-requires-high-fees-for-low-amounts-transactions but I found no information that seemed to fit my situation.  I have never deposited very small amounts (inputs?).

Can someone explain in plain english (I apologize for my ignorance) if the $30 I have currently have in Copay is rendered useless? If so, why (and how do I prevent this from happening in the future) and if not, how do I use these funds.

I am using Copay app on iOS iPhone.
![img_2578](https://user-images.githubusercontent.com/8270293/29809724-122d0aa8-8c52-11e7-91a5-a3be7b43fb49.PNG)

Many thanks for your help!

**Comment by JDonadio** [comment]
@MelisaSingh the error "Not enough funds for fees" is related to the transaction size, not about the amount itself. Each input used increase the transaction size, which means more fee. You can try using the send max amount and see if you are able to spend these funds.

![cursor_y_copay_-_copay_bitcoin_wallet](https://user-images.githubusercontent.com/13851807/29825667-7cc39d4c-8cab-11e7-9b6a-46469040396e.png)

**Comment by audiopathik** [comment]
So, I am in the very same situation and have a balance of 50.74 EUR in my Copay Wallet and there is the orange warning notification "Spending this balance will need significant Bitcoin network fees" below it.

By clicking on it I can find out I received that balance in two seperate transactions and that both amounts have their own bitcoin addresses - one of them making up nearly all my balance (nearly 50 EUR) and the other barely more than 1 EUR.

So according to the articles Javier Donadío linked in the FAQ the reason for this significant amount of Bitcoin network fees is that I have split my balance across multiple bitcoin addresses and Copay can provide the bitcoin addresses for these multiple transactions I received (see Screenshot) I am wondering how can I configure Copay to only use one particular bitcoin adress for payments to decrease the amount bytes that need to be computed by the Bitcoin network?

Since in Copay you can see bitcoin addresses for all the funds you received, how could I transfer all the split up balances into one bitcoin address to decrease network fees?

And one word on all this issue: This is not a very convenient payment method, I was trying to pay 12.41 EUR with this balance, including the fees the webshop charged for the transaction it were 22.43 EUR and the significant Bitcoin network fees Copay added made up 36.51% of the transaction so it were 35.33 EUR instead of 12.41 EUR just because of fees ?!

**Comment by armin884** [comment]
Got the same error as audiopathik described with my bitpay wallet. But i don't know why my balance is split across multiple addresses?  How can i fix that? With my blockchain online wallet i never had such problems but since for making payments to merchants i'm now forced to use a payment protocol compatible bitcoin wallet..
